courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<ul><li>I'd add in point 4.: "at the beginning and at the end of each sentence" (if that's what the function does, and where it adds &lt;start&gt; and &lt;end&gt; if not)</li><li>"subsequent cells" -&gt; "subsequent cells:"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>